docs/framework/unmanaged-api/profiling/cor-prf-rejit-flags-enumeration.md

```diff
@@ -37,7 +37,7 @@ typedef enum
 |`COR_PRF_REJIT_INLINING_CALLBACKS`| Receive `GetFunctionParameters` callbacks for any methods that inline the methods requested to be ReJITted. |  
 
 ## Requirements  
- **Platforms:** See [.NET Core supported operating systems](../../../core/windows-prerequisites.md#net-core-supported-operating-systems).  
+ **Platforms:** See [.NET Core supported operating systems](../../../core/setup/index.md).  
```

Comment:
This needs to go to https://review.docs.microsoft.com/en-us/dotnet/core/setup/dependencies?branch=pr-en-us-15569&tabs=netcore30&pivots=os-windows#supported-operating-systems instead

Applies to all links in the unmanaged-api docs.